### DISCUSSION TITLE
Can't install node-sass on Win10 machine

### DISCUSSION THREAD

**Discussion by LeeBreisacher** [cxw42/TabFern#254] [Q&A]
I need some troubleshooting advice. 
I forked the repo and I'm following the instructions in CONTRIBUTING.md.  I'm on Windows 10. I'm in branch 'master'.
I installed node.js, ran 'npm install -g npx' no problems.
Next is step 3, "Run npm install. This will download all the other packages TabFern and its build process use.", but I get:
===
C:\git\TabFern>npm install

> node-sass@4.9.4 install C:\git\TabFern\node_modules\node-sass
> node scripts/install.js

Downloading binary from https://github.com/sass/node-sass/releases/download/v4.9.4/win32-x64-83_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v4.9.4/win32-x64-83_binding.node":

HTTP error 404 Not Found
===
What am I doing wrong? Suggestions? 

Thanks,

Lee

**Comment by cxw42** [comment]
Thanks for being willing to contribute; sorry you're having trouble!  I found https://github.com/sass/node-sass/issues/2773#issuecomment-566644144 that might help, but I don't know for sure.  You might also try `npm update` --- perhaps 4.9 has aged out or something.  Let me know!
